strings/edit_distance.py

```diff
@@ -1,32 +1,48 @@
 def edit_distance(source: str, target: str) -> int:
     """
-    Edit distance algorithm is a string metric, i.e., it is a way of quantifying how
-    dissimilar two strings are to one another. It is measured by counting the minimum
-    number of operations required to transform one string into another.
+    Calculate the edit distance between two strings.
```

Comment:
Can we change the documentation as we'll the code too at the same time?

Reply:
Is there any rule about it?

Reply:
`You need to opt` for the option on which you're working with >_

Reply:
Got it, I'll close this and pull another one.